updating/updating_a_cluster/updating-cluster-cli.adoc

```diff
@@ -16,14 +16,14 @@ You can update, or upgrade, an {product-title} cluster within a minor version by
 == Prerequisites
 
 * Have access to the cluster as a user with `admin` privileges.
-See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
-* Have a recent xref:../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your update fails and you must xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
+See xref:../../authentication/using-rbac.adoc#using-rbac[Using RBAC to define and apply permissions].
+* Have a recent xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your update fails and you must restore your cluster to a previous state.
```

Comment:
Just checking that this was purposeful to remove this second xref?

Reply:
Correct, this was in response to feedback from @mjpytlak. Thanks for checking!